frontend/src/metabase/parameters/components/WidgetStatusIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import { useCallback } from "react";
+import cx from "classnames";
+import { Icon } from "metabase/ui";
+
+type Props = {
+  name: "close" | "enter_or_return" | "empty" | "chevrondown";
+  onClick?: () => void;
+};
+
+export function WidgetStatusIcon({ name, onClick }: Props) {
+  const classes = cx(
+    "flex-align-right flex-no-shrink",
+    name === "close" && "cursor-pointer",
+  );
+
+  const handleOnClick = useCallback(
+    e => {
```

Comment:
I'm not sure why TS let this one go, but `e` is `any` here.

```suggestion
    event: MouseEvent<HTMLImageElement | SVGElement>
```